Source/RapyutaSimulationPlugins/Public/Core/RRGameSingleton.h

```diff
@@ -103,30 +97,100 @@ class RAPYUTASIMULATIONPLUGINS_API URRGameSingleton : public UObject
     //! And please note that the Sim does not store assets in Project, just to make them accessible among plugins.
     static constexpr const TCHAR* ASSETS_ROOT_PATH = TEXT("/");
     static constexpr const TCHAR* ASSETS_PROJECT_MODULE_NAME = TEXT("Game/RapyutaContents");
-    
+
     /**
-     * @brief Get the Assets Base Path object. 
+     * @brief Get name prefix of asset
+     * @param InDataType
+     * @sa [Asset naming convention](https://docs.unrealengine.com/5.2/en-US/recommended-asset-naming-conventions-in-unreal-engine-projects)
+     */
+    FORCEINLINE static constexpr const TCHAR* GetAssetNamePrefix(const ERRResourceDataType InDataType)
```

Comment:
We should document in the README that these conventions are required.

Reply:
Yeah I added note to doxygen